component/src/test/java/io/siddhi/extension/io/file/FileSourceLineModeTestCase.java

```diff
@@ -1323,7 +1323,7 @@ public void receive(Event[] events) {
     }
 
     @Test
-    public void siddhiFileTestCronSupport() throws InterruptedException {
+    public void siddhiIOFileTestCronSupportForFile() throws InterruptedException {
         log.info("Siddhi IO File test for Cron support");
```

Comment:
Shall we change these descriptions to self explain what are the test cases are for?

Reply:
fixed in 33a0d4ec677b82f844777943dec4b611303e759d